app/src/testDemo/kotlin/com/google/samples/apps/nowinandroid/ui/NiaAppScreenSizesScreenshotTests.kt

```diff
@@ -138,6 +139,11 @@ class NiaAppScreenSizesScreenshotTests {
         TimeZone.setDefault(TimeZone.getTimeZone("UTC"))
     }
 
+    @After
+    fun after() {
+        composeTestRule.activityRule.scenario.close()
```

Comment:
`composeTestRule` being registered as a `Rule`, this `close` mechanism should already be handled.

Reply:
I've checked, and it does seem to be true! Thanks!